### PR TITLE
MCR-2302 add property MCR.ConfigDir

### PR DESCRIPTION
**Pull request by sebhofmann**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2302).

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 86577209de9f018f4dcae8bc43fa8a9d47caef12! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/6183-pull_request-1101

File: mycore-base/src/main/java/org/mycore/common/config/MCRConfigurationDir.java [hunk 1]
```diff
@@ -38,7 +38,9 @@
 
 /**
  * This helper class determines in which directory to look for addition configuration files.
- * 
+ *
+ * The Configuration Directory can be set with the System property <code>MCR.ConfigDir</code>.
```
Comment (kkrebs) [inline mycore-base/src/main/java/org/mycore/common/config/MCRConfigurationDir.java]: The configuration directory can be set with the system property <code>MCR.ConfigDir</code>.

**Review by kkrebs — APPROVED** [review]

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on b8dc4cd7639f7482509c8a95f5a6c6d280fe5e84! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/6193-pull_request-1101

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on b8dc4cd7639f7482509c8a95f5a6c6d280fe5e84! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/6193-pull_request-1101

**Pull request merged**